src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -133,20 +155,26 @@ private static void LogErrors(ScriptHost host)
         }
 
         // Let the existing host instance finish currently executing functions.
-        private async Task Orphan(ScriptHost instance)
+        private async Task Orphan(ScriptHost instance, bool forceStop = false)
```

Comment:
Add a descriptive comment block on this method, explaining the "forceStop" param